scripts/coords.js

```diff
@@ -0,0 +1,70 @@
+
+var _ = require('./constants.js');
+
+
+
+
+// -----------------------------+ private functions +--------------------------------
+
+function LatLon(lat, lon){
```

Comment:
Since the resulting object doesn't have any behaviour, I would just use an object literal instead of newing up an object of this type.